api/app/responses/tests.py

```diff
@@ -985,3 +991,172 @@ def test_response_detail_admin_only(self):
             json=params)
 
         self.assertEqual(response.status_code, 403)
+
+class ResponseExportAPITest(ApiTestCase):
+    def _data_seed_static(self):
+    
+        self.event1 = self.add_event(key='event1')
+        self.event1admin = self.add_user('event1admin@mail.com', is_admin=True)
+        self.user1 = self.add_user('user1@mail.com', user_title='Ms', firstname='Danai', lastname='Gurira')
+
+        application_form = self.create_application_form(self.event1.id)
+        # Section 1, two questions
+        section1 = self.add_section(application_form.id)
+        self.add_section_translation(section1.id, 'en', name='Section1')
+        question1 = self.add_question(application_form.id, section1.id)
+        self.add_question_translation(question1.id, 'en', headline='Question 1, S1')
+        question2 = self.add_question(application_form.id, section1.id)
+        self.add_question_translation(question2.id, 'en', headline='Question 2, S1')
+
+        # Supplementary file included in application upload (e.g. CV) - Section 1
+        question_supp1 = self.add_question(application_form.id, section1.id, question_type = 'file')
```

Comment:
It would be good to also add a multi-file question, since your API does have some specific logic for handling multi-file